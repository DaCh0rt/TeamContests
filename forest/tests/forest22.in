891 992
-7967 4207
-2511 -6415
-9456 -2005
3299 -417
-4643 -6478
-3860 -8200
-4830 -911
-5919 7313
7232 3463
319 4921
-2053 -8313
4710 695
-4649 -6567
-3033 3642
8900 -774
7293 507
-6437 -1770
4632 -260
-6991 -7135
-836 4112
7209 3620
-5520 -5149
-3680 6354
7275 -1847
7087 -8931
-5881 -7835
-8297 8462
-2719 -2163
-1085 -1912
-2539 -6155
-8762 5742
-5596 2703
-167 -2996
-7690 -6448
-4127 -6218
-6327 4319
3803 -4551
5338 -592
5065 1270
-6460 -5660
-6959 -5986
2681 6455
-5952 5955
5602 -1480
-9651 2876
-2220 2197
-4113 6907
1632 4293
-1033 -8062
1788 -4045
458 -3194
3444 -234
-6144 -657
5366 1089
-4161 2004
-1423 7651
-5946 -6185
4189 -1300
9185 -4315
-5873 -1128
-3857 2967
-4096 -9573
2704 -2028
5595 2148
301 -7534
6300 -271
985 814
1729 -6767
-9825 -3279
4846 -2298
-5155 -7385
-3681 7846
8919 -2410
-1033 4166
5355 -2733
2941 -8008
8458 -9346
-7895 918
-6426 6326
6167 1830
-8226 -3443
9279 -4152
-6232 -2915
-1893 9370
5895 5876
6454 5112
-3706 -9073
4704 8809
-2725 4042
-8005 6931
-9697 6841
-6460 2659
-8762 7868
9582 -3465
6452 6481
1836 4584
4389 7939
-3331 8715
-6301 1537
3360 4684
3097 6243
-861 -4468
-7874 2522
-6272 -8672
3299 3420
-3625 4996
-6817 1413
939 590
5629 5878
-2327 -9214
5757 8116
-3186 -1092
-9367 -5599
-8945 -9489
-3307 707
9870 -7774
7368 2199
-3007 -5925
-3337 4396
8989 -300
1965 -3276
3962 176
-6622 -9526
-8067 -8818
5794 -9011
-7259 2873
2036 9458
599 9469
7091 -9809
-5402 9564
8207 2384
-4113 -9759
23 8578
2433 -5210
7448 3746
-8059 -5970
7985 1203
-4184 -3608
-254 7412
9750 -3556
2168 3520
6343 4302
-7289 -1536
6389 6341
5460 6122
-5573 784
-1864 -8954
6652 -7270
-856 -8738
2122 1503
5079 3871
4381 936
6877 -9032
8385 -4809
7516 -381
-6699 9898
-2637 1962
1273 8091
1041 7936
-4775 -6229
4043 -4028
-2066 -4871
-9543 -6006
6320 433
4490 -7034
1079 9748
2670 56
4918 -5460
3204 -5228
3809 -2012
-2979 -238
-5606 -4514
7243 1396
2837 240
-6972 8217
5459 -8558
-378 4140
7244 9965
-8616 -3719
-3131 -2281
-8427 -2905
4627 -8281
-4211 -7151
3477 9574
-8319 7185
-3229 175
-8634 -9965
6145 6723
9766 8009
7075 -9444
-7090 -7486
-2612 -2764
-7140 5710
-3121 -5021
-7217 -6958
-1056 3976
-5690 -2151
511 -755
5858 -8946
-8013 -4333
-1873 -3778
-7934 -158
-7983 -5072
5431 -4946
-5335 6521
-6158 702
-7982 1438
9957 6383
8562 6904
4180 8154
-1459 -9240
9314 -8036
3773 -6476
-4760 -2893
-6185 9685
-2689 6142
-6169 -5190
-5573 -9300
-1449 6016
1000 4491
-4818 -1256
-4509 -9791
-6159 7993
-102 -1118
-3343 -4081
-7123 8197
-7841 -8099
6449 6766
-5498 -4112
-6258 -2424
4534 -4632
-8953 7351
300 2740
-7586 3186
-3483 5834
-9242 -5674
-8253 7002
4905 4674
6906 -615
4749 9876
-9234 -2834
-7522 7450
3760 -7899
1653 -2284
-9001 7767
4935 -3134
1424 -5349
2724 -9637
636 1737
4925 -8808
2666 -9743
-7984 9009
-7977 7385
-5265 -4594
7589 -5009
-1597 7021
-2552 6180
-2699 6565
-4843 6335
-909 -7797
-3857 -8616
7328 -4082
-3190 -2178
-7477 -9376
-1627 -8830
2289 1458
-5123 -1508
1924 8639
-8659 -9617
9312 3176
-1786 -3642
7924 -6367
-2286 3919
2520 -1866
1655 6575
-6676 -2776
-1520 7637
8858 -6970
-4571 -9224
-4783 -7233
259 -1425
2609 9125
-7912 9727
8030 2360
8127 5147
-2679 9333
8355 1791
6922 -5369
4129 4875
-1322 5721
-6935 -2011
-3987 9747
1657 3378
-2519 7556
6271 -4125
5571 6514
6784 -620
6272 -9918
-7452 -2576
3038 -204
8227 -2784
9578 5812
-4967 -5991
7560 -3059
3263 9033
-1958 -6137
3341 5641
-1932 2374
-237 -5625
6487 -228
-3255 -4531
-7162 -9878
-2882 1260
-2379 8075
7440 -9703
7281 4139
5216 -5286
7734 5252
6988 -6381
-4026 169
-7941 -2381
2082 4248
-8164 3469
-5154 -4545
5045 2857
-2840 -437
6783 7298
-2573 1108
6359 -7917
5749 -4435
-3444 -9338
-820 9504
15 -5045
-2491 3486
2467 8273
2164 513
1007 4632
-6272 3089
6090 2477
-5410 4208
-2523 351
-3741 -2691
-574 -837
3102 -2706
-583 -4005
-8050 4836
-2886 2041
-4478 -6793
-2256 7671
-3540 6432
7975 -4959
1672 4240
1720 8927
387 -5663
-466 -3557
-18 -4493
7941 1954
-5332 1116
-3093 -5084
3804 9847
7933 -9101
-8352 -6777
-7625 -1
1064 -5973
4370 1694
-2334 5267
6932 -1496
-350 4237
-7249 -5431
2917 6515
2496 8202
-2598 -9681
1026 1162
4995 1301
4634 -2289
6935 7718
389 -9125
9122 -4018
-5614 252
7537 1598
657 6975
7867 -3813
-5566 -1336
9576 9672
-5089 1997
-8269 7738
5445 -6819
9506 -4745
70 -8528
-4457 4381
5183 -130
4930 7454
8956 5335
-4653 -4781
-169 8087
4636 -1518
-591 -6694
-1838 6838
-6937 7795
1806 -6100
9190 -4878
-7928 -3604
-2144 102
-9062 6370
-2074 8137
-2038 6541
-7884 9632
969 -8674
-1164 5514
-7542 -3643
-5336 -8600
-5144 -2758
3236 -8760
4868 -2160
3889 5063
-450 -2173
5154 8001
-6733 9674
8085 -5154
9537 6149
-2627 -1275
3886 -6338
9074 1582
-2750 8849
-8312 966
-3242 6341
-1187 -2518
-8359 9062
-191 -3591
3829 -7844
-1976 3618
1854 2418
-6240 8316
-6534 -513
3580 -9079
9964 2176
-9037 -5281
8232 3407
4415 -7249
1382 -7341
-4202 3752
-6334 6098
-1506 -9847
4055 3786
2365 -740
-2926 -545
1990 4755
-9476 -1926
1146 1314
-8020 -1216
-6516 2128
3386 -1380
2633 2736
9103 -1748
-1179 -5470
4709 5567
3784 4189
-5547 -1377
-7140 -1350
8841 5972
2375 3611
527 3714
5416 5532
-3934 7703
-6004 -9488
-9978 -1113
4778 -8964
-7642 7559
-7484 1811
-4636 1045
7355 4924
2738 9021
-9657 3833
-2732 6560
-802 6886
-3615 -2375
-9964 3791
-9893 -7972
8333 6660
-193 -9322
6264 114
-584 8280
-3993 -958
2800 294
-9693 -2885
5525 9177
9375 -9847
7759 -5912
-1741 2764
-1583 -6928
-4569 -3743
8171 -3349
-6363 -1275
3313 7702
9272 -3535
-8682 -3927
1030 1098
8759 -4600
-6058 2311
7454 -9006
-6775 4839
9326 -2610
3278 7620
-5843 8518
8317 7279
5443 -5170
-3782 2136
9940 -4912
-8070 1529
2396 8856
1180 9182
-8603 -2452
-5447 9539
-8495 -5509
-2533 -2974
1523 7563
-9476 -5831
-7738 -1828
-1985 -1887
7082 -5704
-2661 2930
6343 1604
-423 8002
-1855 7355
222 6202
-6275 7597
-4198 -7892
-9519 -2942
7907 -4762
-2833 -4635
9666 9095
-7820 9855
1523 7535
-1645 1609
3232 -744
-1288 -3324
-6568 2172
-2416 -8519
8980 3728
5391 4686
9414 -8735
6002 3608
-4234 -1916
2719 4178
-8171 -7226
6726 -4653
-1332 1577
6384 1443
4575 5459
-6018 -4399
1967 -3622
9426 -6503
-5770 -6844
4166 -8076
-5076 -3099
-5049 8510
4997 2595
-2839 -6419
-5120 1810
2458 -3539
-9522 6676
7192 9012
-4793 -1337
6896 4731
-8082 -4992
-6892 -7640
-9025 9855
-2074 5551
454 -5573
-7965 -3970
4533 -5219
-6159 -4602
3386 -3761
-312 9942
-3369 -7825
-3938 6569
-3419 -3031
-1020 7509
-1447 -5906
-201 -5732
9028 8316
-1479 -4868
-7952 4675
-1297 -441
-350 664
3338 -2147
6589 -3488
1228 5901
-2637 -7258
6894 4084
5583 -2778
7460 3544
6035 9215
4311 5824
8267 -142
-8848 5188
-663 388
-5636 262
1069 8508
5841 2626
-5926 -8062
-6089 -668
9672 -7473
8392 4592
7253 -4282
3229 6730
2598 -7708
2104 2422
-89 -9403
-654 -2612
939 5800
-2923 -7314
-6947 1619
-398 -1992
-7983 8198
-1371 -1882
750 3032
-5478 -5640
903 -6702
5744 -5799
-2045 9662
-5210 -2951
9001 -3115
2441 -7361
617 -3627
-381 1941
3959 -5775
-4109 -5159
-6606 9630
-3075 8047
-7747 -5241
-6530 -8795
3367 3847
-7445 1270
1925 -1856
-3020 -9492
-6775 -7634
3653 -3498
-4727 -551
-5705 -8448
-814 2778
9724 6636
-5884 -3625
-6560 9312
-9463 -7663
1683 2400
209 1905
7117 1224
8627 -6135
-3663 7696
-6064 -402
-9953 6110
5072 5324
9780 2651
2128 821
2850 -4045
-4984 -8734
-8890 1972
8272 2016
-2834 -5412
-8635 7122
2457 5553
-3156 601
5902 794
-6955 -6190
6721 -8740
7908 5386
-5010 -5689
6349 8210
5969 5996
7258 9113
8632 -6872
-7033 8230
9935 5241
-6003 -1437
3258 -2060
-2994 6580
9205 -1021
4810 2747
-5613 -6792
-3330 -8764
5629 -8626
2435 -8363
-3955 8997
-1297 3346
-6727 -9425
9316 1519
6597 642
1431 2259
9752 9166
-1335 275
-7141 -5251
3561 -2545
-8261 -9152
-5607 5521
-3833 -7357
-1190 -4702
4964 1971
-8406 163
-7025 -1185
1414 -9812
49 8163
-8149 -2161
9812 735
-2246 9981
-9570 1308
-2540 1382
2645 4524
-368 -6946
9185 301
-4961 -9706
3909 583
-5653 2442
-3877 3835
8979 154
-1983 -5727
4701 10
4008 7122
-223 8488
5500 -9167
9906 -1099
9854 3360
9917 -5393
-9514 8573
7372 -3950
-6049 -7497
7223 -7532
7407 -9984
-1477 -4978
-974 7983
-5652 -1342
250 8785
-245 2735
2462 -2771
-8166 7938
-5454 6487
-5882 -4741
-3734 -9141
2222 -9142
-1555 8546
5878 8595
7694 -5651
-8420 -2794
-5904 -106
1273 4941
3894 1357
9997 690
-2857 8916
-7784 -5166
-7328 -3030
-1041 -5298
8794 -7795
-1724 9213
4444 5682
-6354 -7791
-6968 -8513
8955 -6604
-4523 -1516
-6061 -7509
4891 563
-9493 6128
-1794 -1474
-8590 -1473
8178 -8606
8550 -9300
3818 -6815
-1645 4146
1873 5754
-3810 591
524 1974
9113 -8132
6342 -4714
-9699 -2664
-6610 -1693
9211 -613
4307 -5285
3236 -7590
-9790 -8194
-8122 848
-8701 -3139
-2237 5216
-3889 3847
8115 -5641
2528 -6019
-6255 6080
-9907 -8080
-5488 6840
9755 -351
-8714 -1589
-2309 758
-3768 -9212
-6975 1410
1211 -523
828 -1760
5264 689
3436 -3426
-3283 -5116
-5640 -3412
-9707 6863
-8216 9171
-574 4381
5561 -3885
8723 -5652
-1186 -3074
5077 -3481
-3628 -742
9339 3110
1748 3011
7434 -2708
3849 -3807
-798 7989
-5207 -7396
1518 8695
-2628 -2560
-1108 -2605
3359 -1452
7193 6753
9962 6720
6266 7752
8415 -5053
-7216 3288
-410 -5750
3963 -9124
2425 -7480
4471 -3127
3235 -8019
6032 1505
-9064 8923
-9571 -5157
-3396 4683
5102 -5189
510 -676
-5358 -3118
4268 7665
-6904 -8210
5045 -8720
5001 2613
-9041 -1590
7222 2648
7636 4353
7254 9844
-3101 -8804
-8974 3384
-894 9102
-4315 6222
-64 5012
-249 -2483
-5685 1056
-7794 -835
485 -1119
-7533 -1515
-8943 -8114
-4165 -2684
505 -2434
8939 -7839
-2367 6940
-9280 750
-1216 6878
-5279 6117
-7947 -5285
3414 -4749
7662 5423
1272 8324
-3699 -5371
1104 4831
8912 5227
-7718 5390
4232 -767
7874 1781
-5483 242
-4573 -360
-3867 1324
-9930 4329
5490 3895
-8474 508
4309 -8596
4278 5117
3993 -6641
-1925 -8184
8503 -7130
-1474 -3743
3652 810
3790 -8178
1709 5898
-549 5046
3565 1022
703 -6707
-2374 8498
983 -6883
7022 6329
-2872 -8334
