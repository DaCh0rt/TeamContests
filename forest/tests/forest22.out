82823.38 471353303.17
